packages/bbui/src/Typography/Heading.svelte

```diff
@@ -6,11 +6,17 @@
   export let textAlign: string | undefined = undefined
   export let noPadding: boolean = false
   export let weight: "light" | "heavy" | "default" = "default"
+  export let titleColor: string | undefined = undefined
 </script>
 
 <h1
   data-testid="typography-heading"
-  style={textAlign ? `text-align:${textAlign}` : ``}
+  style={[
+    textAlign ? `text-align: ${textAlign}` : "",
+    titleColor ? `color: ${titleColor}` : "",
+  ]
+    .filter(Boolean)
```

Comment:
Sneaky little trick. Very terse, I like it. Although if we can remove the title colour setting then we can also undo changes to this file, but I'll let that be informed by whether or not a unique title colour was part of the customer request. If we're keeping this, then we at least need to rename this setting to `color` rather than `titleColor`.